src/views/CoCurricularTranscript/index.tsx

```diff
@@ -36,14 +44,155 @@ const CoCurricularTranscript = () => {
     loadTranscript();
   }, [profile]);
 
-  if (loading || loadingProfile) {
+  if (loadingProfile) {
     return <GordonLoader />;
   }
 
   if (!profile) {
-    return <GordonUnauthorized feature={'your experience transcript'} />;
+    const participation1: Participation = Participation.Member;
+    const session1: MembershipHistorySession = {
+      MembershipID: 6547,
+      SessionCode: '202306',
+      Participation: participation1,
+    };
+    const activity1: MembershipHistory = {
+      ActivityCode: '360',
+      ActivityDescription: '360.gordon.edu',
+      ActivityImagePath: '',
+      Sessions: [session1],
+      LatestDate: '2023-06-07T00:00:00',
```

Comment:
I would love to, but can we really put a relative date rather than a specific date and time since the API should bring back the specific date and time for the web to display?

Reply:
I would think that would be fine.